packages/main/src/plugin/agent-workspace/agent-workspace-manager.spec.ts

```
@@ -23,10 +23,12 @@ import { beforeEach, describe, expect, test, vi } from 'vitest';
 import { parse as parseYAML } from 'yaml';
 
 import type { IPCHandle } from '/@/plugin/api.js';
+import type { CliToolRegistry } from '/@/plugin/cli-tool-registry.js';
 import type { Proxy } from '/@/plugin/proxy.js';
```

Biome (2.4.9)

[error] Do not shadow the global "Proxy" property.

(lint/suspicious/noShadowRestrictedNames)